json/Side_Text.txt

```diff
@@ -28684,175 +28684,175 @@
 		"jp_text": "気合い入ってるわね。<%br>でも、いつまで続くかしらね……？",
-		"tr_text": "",
+		"tr_text": "Ooh, I like your enthusiasm.<%br>I wonder how long that'll last...?",
 		"fileID": 28
 	},
 	{
 		"eventNo": "16",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "いいえっ！　意地でも耐え抜きますっ！",
-		"tr_text": "",
+		"tr_text": "No! I'll make it through this, no matter what!",
 		"fileID": 28
 	},
 	{
 		"eventNo": "17",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "だいじょうぶか……？",
-		"tr_text": "",
+		"tr_text": "You really OK there...?",
 		"fileID": 28
 	},
 	{
 		"eventNo": "0",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "はーっ、はーっ……",
-		"tr_text": "",
+		"tr_text": "＊pant＊, ＊pant＊...",
 		"fileID": 28
 	},
 	{
 		"eventNo": "1",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "こ、こ、こ、こんなのは、もう……<%br>やりたくないれす……",
-		"tr_text": "",
+		"tr_text": "We, we, we, we're done...<%br>Make it stop...",
 		"fileID": 28
 	},
 	{
 		"eventNo": "2",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "あら耐えちゃった。<%br>絶対途中でへばると思ってたのに。",
-		"tr_text": "",
+		"tr_text": "Aw, you held out. And here I thought you'd<%br>give up before we were halfway through.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "3",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "もしかしてジェネちゃんって鈍感？<%br>それとも、慣れてないから逆に強いのかしら。",
-		"tr_text": "",
+		"tr_text": "Do you just have low sensitivity? It should have<%br>felt extra-strong, since it's your first time.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "4",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "と、とりあえず、元に戻してください……",
-		"tr_text": "",
+		"tr_text": "C-can you please just turn it off...",
 		"fileID": 28
 	},
 	{
 		"eventNo": "5",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "なんにしても、負けは負けね。",
-		"tr_text": "",
+		"tr_text": "Well, a loss is a loss.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "6",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "約束通り、リーダーは一旦諦めて<%br>しばらくはからかうだけにしておくわ。",
-		"tr_text": "",
+		"tr_text": "As promised, I'll give up on Leader.<%br>I'll stick to teasing from now on.",
 		"fileID": 28
 	},
 	{
 		"eventNo": "7",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "からかうのはやめないんですね……",
-		"tr_text": "",
+		"tr_text": "I thought you'd give up on that too...",
 		"fileID": 28
 	},
 	{
 		"eventNo": "8",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "ところで、ジェネちゃん。",
-		"tr_text": "",
+		"tr_text": "By the way, Gene...",
 		"fileID": 28
 	},
 	{
 		"eventNo": "9",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "なんでしょう？",
-		"tr_text": "",
+		"tr_text": "What is it?",
 		"fileID": 28
 	},
 	{
 		"eventNo": "10",
 		"jp_name": "ブルージーレクイエム",
 		"tr_name": "Bluesy Requiem",
 		"jp_text": "チームのためかリーダーのためか<%br>知らないけれど、よく耐えたわね。",
-		"tr_text": "",
+		"tr_text": "I don't know what you were thinking of to stay<%br>strong, but watching you endure the pleasure...",
```

Comment:
`I don't know if you were doing it for the team or doing it for Leader's sake, but you've handled that pretty well...`